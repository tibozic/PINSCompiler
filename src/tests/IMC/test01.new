Global: size[4],label[x]: 
FRAME [f]: level=1,locals_size=0,arguments_size=0,parameters_size=8,size=8
SEQ:
  EXP:
    BINOP ADD:
      NAME: x
      CONSTANT: 5
  MOVE:
    NAME: {FP}
    BINOP ADD:
      NAME: x
      CONSTANT: 5
