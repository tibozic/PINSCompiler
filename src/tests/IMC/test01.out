Global: size[4],label[x]: 
FRAME [f]: level=1,locals_size=0,arguments_size=0,parameters_size=8,size=8
EXP:
  BINOP ADD:
    NAME: x
    CONSTANT: 5
